balancer-js/src/modules/swaps/joinAndExit.integration.spec.ts

```diff
@@ -113,21 +115,23 @@ describe('join and exit integration tests', async () => {
     // SwapTypes.SwapExactIn,
     '10' // 10 bsp = 0.1%
   );
-  await testFlow(
-    'swap, join > swap, mulithopswap - WETH[Swap]auraBal, WETH[join]BPT[Swap]auraBAL, WETH[Swap]wstETH[Swap]auraBal',
-    [
-      BAL_WETH,
-      AURA_BAL_STABLE,
-      GRAVI_AURA,
-      B_50auraBAL_50wstETH,
-      B_stETH_STABLE,
-    ],
-    parseFixed('12.3', 18).toString(),
-    ADDRESSES[networkId].WETH,
-    ADDRESSES[networkId].auraBal,
-    '10', // 10 bsp = 0.1%
-    15773550
-  );
+
+  // Removed test that started failing after updating to a more recent blockNumber
```

Comment:
I had to update to a more recent block due to relayerV5 and this test started failing. I tried figuring out why, but it was taking too long and I decided it would be better to check before spending more time on it.
@johngrantuk - this was the only test with a custom blockNumber - do you remember why that was the case?

Reply:
I don't actually remember. I wanted to work on refactoring joinAndExits a bit next week so can investigate then. Should be ok to ignore for now.